components/ui/AppDataTableSection.vue

```diff
@@ -8,15 +8,17 @@
   </section>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Prop } from 'vue-property-decorator'
+<script setup lang="ts">
 import { TableRowElement } from '~/components/ui/AppDataTable.vue'
 
-@Component
-export default class AppDataTableSection extends Vue {
-  @Prop({ type: String, required: true }) sectionTitle!: string
-  @Prop({ type: Array, default: () => [] }) dataTableColumns!: string[]
-  @Prop({ type: Array, default: () => [] }) dataTableElements!: TableRowElement[]
+interface Props {
+  dataTableColumns?: string[]
```

Comment:
Yes, that prop needs to be mandatory because is has to be passed to `AppDataTable`, which requires it.

Changed in https://github.com/Qiskit/qiskit.org/pull/2935/commits/af67afaf14b05f009fd71f7cee64c2ba593bbc7a